cassandane/Cassandane/Cyrus/TestCase.pm

```diff
@@ -1079,6 +1170,16 @@ sub _start_instances
                 if ($self->{_want}->{adminstore});
         }
     }
+
+    my $user = Cassandane::Cyrus::TestCase::User->new({
+        username => 'cassandane',
+        password => 'cassandane',
+        instance => $self->{instance},
+    });
+
+    $self->{default_user} = $user;
+
```

Comment:
A test can have multiple instances running, and a user belongs to an instance...

Mapping a test's default_user to a user on the test's default instance makes some ergonomic sense, but it implies things are simpler than they are.  Perhaps default_user should be per-instance?  I dunno.  If you already know how to write multi-instance tests, you can still do so.  But if you don't know that it's even possible, discoverability is lacking with this API.

Reply:
Yeah, I agree, I don't think this is a blocker, unless you thought about it and decided that it was, which you did and didn't.

(And we can always change it later if our ergonomic needs change.)